doc/glossary.rst

```diff
@@ -27,17 +27,17 @@
       hemoglobin.  Oxy- and deoxy- hemoglobin have different magnetic
       properties.  This in turn leads to a change in MRI signal that
       can be detected by collecting suitably sensitive MRI images at
-      regular short intervals during the blood flow chance.  See the
-      the `wikipedia FMRI`_ article for more detail.
+      regular short intervals during the blood flow chance.
```

Comment:
Aha - you fix is good - but there's another mistake I made 'chance' for 'change'.